docs/api/field_type_reference.md

```diff
@@ -1501,6 +1501,157 @@ new FieldType\Value([
 ]);
 ```
 
+### GraphQL Field Type operations
+
+To get a Matrix Field Type, you will need to specify a Content ID, a Content Type, and a Field Type. 
+The types used for input and returned are named after the Type and the Field: 
+
+- `{TypeIdentifier}{FieldIdentifier}RowInput`
+- `{TypeIdentifier}{FieldIdentifier}Row`
+
+The example below shows GraphQL query for a List Content Type (Content Type with a Matrix Field Type added), that has two fields:
+
+- `name`: `ezstring`
+- `ingredientList`: `ezmatrix` with two columns: `ingredient` and `quantity`
+
+```
+{
+  content {
+    #provide content ID, location ID or remote ID
+    list(id: 123) {
+      name
+      ingredientList {
+        ingredient
+        quantity
+      }
+    }
+  }
+}
+```
+
+The Type returned for the Matrix Field exposes columns defined in the Field definition:
+
+```
+{
+  "data": {
+    "content": {
+      "list": {
+        "name": "Cake ingredients",
+        "ingredientList": [
+          {
+            "ingredient": "Butter",
+            "quantity": "200 grams"
+          },
+          {
+            "ingredient": "Sugar",
+            "quantity": "100 grams"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+**Query for the Field Type and Field definition's details**
```

Comment:
We may want to add a small paragraph explaining why you want to do that (you want to do that either for a) as a developer, inspecting the existing types b) reading details about the content types and their fields in the context of a backoffice like application.

Note that this is not specific to this field type. What is specific for this type is that you can ask for the list of columns, and that they have a name and identifier.